sklearn/tests/test_multioutput.py

```diff
@@ -336,3 +341,123 @@ def test_multi_output_exceptions():
     y_new = np.column_stack((y1, y2))
     moc.fit(X, y)
     assert_raises(ValueError, moc.score, X, y_new)
+
+
+def test_classifier_chain_fit_and_predict():
+    """Fit classifier chain and verify predict performance"""
+    X, Y = datasets.make_multilabel_classification(n_samples=10000,
```

Comment:
That is a very good point about `make_multilabel_classification` not having covariances. I was wondering about that. You're right that it doesn't make much sense to use such a dataset for performance tests. I could switch to using the yeast dataset for all of these tests.